airflow/models/dag.py

```diff
@@ -752,6 +752,10 @@ def get_task_instances(
 
     @property
     def roots(self):
```

Comment:
Can we add a python doc here as well: `Tasks that have no parents. These are the roots of the DAG.`